Formattng all Python files with Black (+ Enforce) (#3071)
scality/metalk8s: docs/developer/development/commit.rst

Commit Best Practices
---------------------

> Pre-commit hook

Reviewer: ```suggestion
Pre-commit hooks
```

~~~~~~~~~~~~~~~

> Some pre-commit hook are define to do some linting check and also to format all

Reviewer: ```suggestion
Some pre-commit hooks are defined to do some linting checks and also to format all
```

Python code automatically.

> Those checks are also done in the CI so make sure it succeed.

Reviewer: ```suggestion
Those checks are also run in the CI pre-merge test suite to enforce code linting.
```

> To run pre-commit automatically when you commit with git just install
> pre-commit ``pip install pre-commit``, and then install the hook
> ``pre-commit install``.

Reviewer: ```suggestion
To enable pre-commit hook to run automatically when committing, install it as follows:

.. code-block:: shell

   pip install pre-commit
   pre-commit install
```

You can skip this pre-commit hook on a specific commit
``git commit --no-verify``.

> To run pre-commit manually just use tox directly ``tox -e pre-commit``.
> It's also possible to run only a specific hook (e.g. for pylint
> ``tox -e pre-commit pylint``).

Reviewer: ```suggestion
To run pre-commit manually, use tox:

.. code-block:: shell

   tox -e pre-commit

It is also possible to run only a specific hook (e.g. for pylint
``tox -e pre-commit pylint``).
```
